templates/confidenceSlider.jsx

```diff
@@ -0,0 +1,108 @@
+import Adapt from 'core/js/adapt';
+import React from 'react';
+import { classes, templates } from 'core/js/reactHelpers';
+
+export default function ConfidenceSlider (props) {
+  const {
+    isInteractive,
+    _canShowMarking,
+    _isInteractionComplete,
+    _isEnabled,
+    _isCorrect,
+    labelStart,
+    labelEnd,
+    _userAnswer,
+    _scaleStart,
+    _scaleEnd,
+    _marginDir
+  } = props;
+
+  const shouldShowMarking = isInteractive() && _canShowMarking;
```

Comment:
https://github.com/adaptlearning/adapt-contrib-core/blob/759b768057ef3d88d83328121fdec25f1a96bb93/js/models/questionModel.js#L368-L374

You should be able to use `_shouldShowMarking` directly from the model now.